.project

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>rt</name>
-	<comment>Project rt created by Buildship.</comment>
+	<name>jfx</name>
```

Comment:
this may cause issues with internal setup at oracle

Reply:
We do still clone the repo into a directory named `rt` when doing a closed build, but I doubt this change will cause problems. Maybe Andy can check? As Nir mentioned, people can (and do) name the root directory whatever they want and it doesn't seem to matter to Eclipse other than it looks a little odd. I agree with the general point that it might be nice to match the default name, but it doesn't matter one way or the other to me.

Reply:
I don't see how a closed build will be affected by Eclipse files.

Reply:
Nor do I. The build doesn't care, and if a developer clones the repo into `rt` after the name is changed to `jfx` it will be no different than someone who clones the repo into, say, `javafx` or `jfx20` or ...

Reply:
the closed repo is also called jfx, but I withdraw my objection.

If one were to create an eclipse project in the closed repo (if only to use egit UI, for instance), that project must be named differently (closed-jfx for instance) because eclipse cannot have two projects with the same name in the same workspace.  therefore, we can rename the open project to jfx.